manifests/o/OpenMW/OpenMW/0.47.0/OpenMW.OpenMW.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: OpenMW.OpenMW
+PackageVersion: 0.47.0
+PackageLocale: en-US
+Publisher: OpenMW Council
+PackageName: OpenMW
+License: GPL-3.0
+ShortDescription: OpenMW is an open-source open-world RPG game engine that supports playing Morrowind.

```

```suggestion
ShortDescription: OpenMW is an open-source open-world RPG game engine that supports playing Morrowind.
InstallationNotes: You need to own the game for OpenMW to play Morrowind.
```
Show a helpful message after installation is complete.